runtime/src/in_mem_accounts_index.rs

```diff
@@ -832,7 +836,7 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
     }
 
     /// return true if 'entry' should be removed from the in-mem index
```

Comment:
nit: I think leaving "removed" here is good / matches other places; however, I think we should update the `remove` variable in this function to `evict`